compiler/damlc/daml-doc/src/DA/Daml/Doc/Render/Output.hs

```diff
@@ -75,8 +75,16 @@ instance RenderDoc TemplateDoc where
             , fieldTable td_payload
             , RenderList (map renderDoc td_choices)
             ]
+        , RenderBlock $ mconcat
```

Comment:
can we get a test that checks the rendered output via the expected markdown & rst files?

Reply:
I'll add a golden test.